docs/calamari/DevelopOnDolphin.md

```diff
@@ -1,14 +1,37 @@
 # 🛠 Develop on Dolphin
 
 ## Introduction
-This guide provides step-by-step instructions to set up a Calamari development node to test your applications on the public testnet named Dolphin. A Calamari development node acts as your personal development environment to build and test applications on the Calamari Network. It eliminates the need for a relay chain and creates blocks automatically when transactions are received.
+This guide provides step-by-step instructions to set up a Calamari development node to test your applications on the public testnet named Dolphin. A Calamari development node acts as your personal development environment to build and test applications on the Calamari Network.
```

Comment:
the recommended way is to NOT have a local node when developing an app and going through SDK (please link)  or connecting polkadot.js API to dolphin for testing.

Reply:
oops, corrected it.